docs/src/main/paradox/getting-started.md

```diff
@@ -0,0 +1,107 @@
+# Getting Started Guide
+
+By now you should understand the fundamental concepts of how a Projection works by reading about its @ref:[use cases](use-cases.md).
+This guide will briefly describe the basic components of a Projection and instruct you step-by-step on how to build a functioning application.
+The example used in this guide is based on a more complete application called `akka-sample-cqrs` that's found in [akka-samples](https://github.com/akka/akka-samples/). 
+It builds a full @ref:[CQRS](use-cases.md#command-query-responsibility-segregation-cqrs-) (Command Query Responsibility Segregation) ES (Event Sourcing) system using a combination of features from the akka toolkit.
+
+## Part 1: Setup your application
+
+Add the Akka Projections core library to a new project.
+This isn't strictly required, because as we add other dependencies in the following steps it will transitively include core as a dependency, but it never hurts to be explicit.
+
+@@dependency [sbt,Maven,Gradle] {
+  group=com.lightbend.akka
+  artifact=akka-projection-core_$scala.binary.version$
+  version=$project.version$
+}
+
+Define the event type protocol that will represent each `Envelope` streamed from the Source Provider.
+Create a typed @apidoc[akka.actor.typed.ActorSystem] for various Projections implementations to use.
+
+Scala
+:  @@snip [ShoppingCartApp.scala](/examples/src/test/scala/docs/guide/ShoppingCartApp.scala) { #guideSetup }
+
+## Part 1: Choosing a Source Provider
+
+A @apidoc:[SourceProvider] will provide the data to our projection. In Projections data is referenced as an `Envelope`, but it could reference an event, a message from a queueing system, or a record from a database table.
+There are several supported Source Provider's available (or you can build your own), but in this example we will use the @ref:[Event Sourced Source Provider](eventsourced.md).
```

Comment:
```suggestion
There are several supported Source Provider's available (or you can build your own), but in this example we will use the @ref:[Akka Persistence `EventSourced` Source Provider](eventsourced.md).
```
Specify that it is `Akka Persistence`'s. Also, should that be `EventSourced`, one word? I'm not sure, TBH. the snippet in [`eventsourced-.md`](https://doc.akka.io/docs/akka-projection/current/eventsourced.html#sourcedprovider-for-eventsbytag) refers to the class `EventSourcedProvider` so it's a bit confusing.